src/Interpreters/DDLWorker.cpp

```diff
@@ -660,8 +670,8 @@ void DDLWorker::processTask(DDLTaskBase & task, const ZooKeeperPtr & zookeeper)
     active_node->setAlreadyRemoved();
 
     task.createSyncedNodeIfNeed(zookeeper);
-    task.completely_processed = true;
     updateMaxDDLEntryID(task.entry_name);
+    task.completely_processed = true;
```

Comment:
This is slightly changes the behavior nodes cleanup, but it should not be a problem, since the task had been already processed anyway.